src/api/app/views/webui2/webui/users/subscriptions/index.html.haml

```diff
@@ -1,32 +1,32 @@
 - @pagetitle = 'Notifications'
 
-.card
-  .card-header
-    %h5 Notifications
+.card.mb-2
+  .card-header.d-flex.justify-content-between
+    %h3 Notifications
   .card-body
     = form_tag(user_notifications_path, method: :put, id: 'notification_form') do
       - unless @groups_users.empty?
-        .card-block
-          %h3 Groups
-          %p You will receive emails from the checked groups
-          - @groups_users.each do |group_user|
-            .custom-control.custom-checkbox.custom-control-inline
-              = check_box_tag group_user.group, '1', group_user.email, class: 'custom-control-input', id: "checkbox-#{group_user.group}"
-              = label_tag group_user.group.title, nil, class: 'custom-control-label', for: "checkbox-#{group_user.group}"
-        %br
-      .card-block
-        %h3 Events
-        %p Choose from which events you want to get an email
-        %ul.list-group.list-group-flush
-        #subscriptions-form
-          = render partial: 'webui/subscriptions/subscriptions_form'
+        .card.mb-2
+          .card-body
+            %h4.card-title Groups
+            %p You will receive emails from the checked groups
+            - @groups_users.each do |group_user|
+              .custom-control.custom-checkbox.custom-control-inline
+                = check_box_tag group_user.group, '1', group_user.email, class: 'custom-control-input', id: "checkbox-#{group_user.group}"
+                = label_tag group_user.group.title, nil, class: 'custom-control-label', for: "checkbox-#{group_user.group}"
+      .card
         .card-body
+          %h4.card-title Events
+          %p Choose from which events you want to get an email
+          %ul.list-group.list-group-flush
+          #subscriptions-form
+            = render partial: 'webui/subscriptions/subscriptions_form'
```

Comment:
There isn't any margin anymore between the checkbox fields and the button

Reply:
Done. I moved the button to the parent card. I think int makes more sense.

![screenshot from 2019-02-27 10-31-34_user_notifications_button](https://user-images.githubusercontent.com/24919/53480167-e857cc80-3a7a-11e9-910d-ca5a23dab8ab.png)